input/blog/2018-01-17-cake-v0.25.0-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Cake v0.25.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.25.0 of Cake has been released.
+
+The first release of 2018 is here! It contains mostly improvements and bug fixes, but it also has one potential breaking change. 
+
+The most notable change in this release is that we no longer use the NuGet.exe CLI tool when installing `addins`, `tools`, `modules` and `scripts` from NuGet sources in our [preprocessor directives](/docs/fundamentals/preprocessor-directives).
+Now we instead default to the in-process NuGet client introduced in Cake [v0.22.0](/blog/2017/09/cake-v0.22.0-released). This will not only fix many previous issues people seen with Cake's preprocessor NuGet handling, but also enable us to simplify boostrappers and pave the way to properly solve addin dependencies. It will also make make pure .NET Core environments work out of the box with `Cake.CoreCLR`.
```

Comment:
"people *have* seen"

Reply:
Also missed the first time: "make make pure"

Reply:
I'll fix.